openra/fakes/map_file_repository.py

```diff
@@ -0,0 +1,28 @@
+from typing import List
+from fs.tempfs import TempFS
+
+from openra.classes.file_location import FileLocation
+from openra.classes.release import Release
+from openra.services.engine_file_repository import ExceptionEngineFolderNotFound
+from openra.services.map_file_repository import ExceptionMapFolderNotFound
+
+
+class FakeMapFileRepository:
+
+    map_exists:bool
+
+    def __init__(self):
+        self.map_exists = True
+
+    def get_oramap_path(self, map_id:int):
+        if not self.map_exists:
+            raise ExceptionMapFolderNotFound(TempFS(), map_id, 'fake')
+
+        temp_fs = TempFS()
+        file = 'sample.oramap'
+        temp_fs.touch(file)
+        return FileLocation(
+            TempFS(),
```

Comment:
Why create a new one here?

Reply:
Yea that doesn't make sense

Reply:
Ok I've updated this now. I don't think that the parts that were being tested with this actually relied on it existing so it wasn't flagging up any issues.